plugins/modules/packaging/language/pear.py

```diff
@@ -51,6 +57,30 @@
     name: pecl/json_post
     state: present
 
+# Install pecl package with expected prompt
+- pear:
+    name: pecl/apcu
+    state: present
+    prompts:
+        - (.*)Enable internal debugging in APCu \[no\]
+
+# Install pecl package with expected prompt and an answer
+- pear:
+    name: pecl/apcu
+    state: present
+    prompts:
+        - (.*)Enable internal debugging in APCu \[no\]: "yes"
+
+# Install multiple pear/pecl packages at once with prompts. Prompts will be processed on the same order as the packages order
```

Comment:
```suggestion
- name: Install multiple pear/pecl packages at once with prompts.
  # Prompts will be processed on the same order as the packages order
```
Here I would put the main description into `name:` and add the rest as comments.